src/main/java/org/zalando/nakadi/util/AuthorizationUtils.java

```diff
@@ -0,0 +1,79 @@
+package org.zalando.nakadi.util;
+
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.EventTypeAuthorization;
+import org.zalando.nakadi.domain.SubscriptionBase;
+import org.zalando.nakadi.exceptions.InternalNakadiException;
+import org.zalando.nakadi.exceptions.runtime.AccessDeniedException;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationAttribute;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationService;
+import org.zalando.nakadi.plugin.api.authz.Resource;
+import org.zalando.nakadi.plugin.api.authz.Subject;
+import org.zalando.nakadi.repository.EventTypeRepository;
+import org.zalando.nakadi.security.Client;
+
+import java.util.List;
+import java.util.Optional;
+
+public class AuthorizationUtils {
```

Comment:
I thought this class is responsible for validating some parameters and not actually implementing code that performs authorization.